workflows4s-example/src/main/scala/workflows4s/example/docs/InMemoryRuntimeExample.scala

```diff
@@ -19,7 +20,9 @@ object InMemoryRuntimeExample {
     import MyWorkflowCtx.*
     val workflow: WIO.Initial                               = ???
     val knockerUpperAgent: KnockerUpper.Agent[MyWorkflowId] = ???
-    val runtime: InMemoryRuntime[Ctx, MyWorkflowId]         = InMemoryRuntime.default(workflow, InitialState(), knockerUpperAgent)
+    val runtime: InMemoryRuntime[Ctx, MyWorkflowId]         = InMemoryRuntime
+      .default(workflow, InitialState(), knockerUpperAgent)
```

Comment:
There is a nasty bug here that took me a while to pinpoint. We don't pass a clock here (neither before nor after your changes). I have no idea how it worked before, but now the tests broke without it.